### PR TITLE
Fix clippy warning in the gateway

### PR DESCRIPTION
**Pull request by leovalais**

### CODE REVIEW

File: gateway/src/config.rs
Comment (woshilapin) [inline gateway/src/config.rs:108]: If we really do want to panic, maybe `#[allow(clippy::whatever_it_is)]` is better in this case?

Comment (leovalais) [reply gateway/src/config.rs:108]: I'm fine with both as long as the warning disappears from my editor ;). I'll let the gateway maintainers decide which one they prefer.

Comment (Khoyo) [reply gateway/src/config.rs:108]: The warning is https://rust-lang.github.io/rust-clippy/master/index.html#/expect_fun_call

The point isn't to prevent the panic, it's to prevent the unnecessary call to format in the happy case.

**Review by Khoyo — APPROVED** [review]

**Pull request merged**
